config/sql/se/Ebooks.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE `Ebooks` (
+  `EbookId` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Identifier` varchar(511) NOT NULL,
+  `Created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `Updated` timestamp NOT NULL,
+  `WwwFilesystemPath` varchar(511) NOT NULL,
+  `RepoFilesystemPath` varchar(511) NOT NULL,
+  `KindleCoverUrl` varchar(511) NULL,
+  `EpubUrl` varchar(511) NULL,
+  `AdvancedEpubUrl` varchar(511) NULL,
+  `KepubUrl` varchar(511) NULL,
+  `Azw3Url` varchar(511) NULL,
+  `DistCoverUrl` varchar(511) NULL,
+  `Title` varchar(255) NOT NULL,
+  `FullTitle` varchar(255) NULL,
+  `AlternateTitle` varchar(255) NULL,
+  `Description` varchar(511) NOT NULL,
+  `LongDescription` text NOT NULL,
+  `Language` varchar(10) NULL,
+  `WordCount` int(10) unsigned NOT NULL,
+  `ReadingEase` float NOT NULL,
+  `GitHubUrl` varchar(255) NULL,
+  `WikipediaUrl` varchar(255) NULL,
+  `Released` datetime NOT NULL,
+  `MetadataModified` datetime NOT NULL,
```

Comment:
What will this column be used for?

Reply:
I'm going to rename the existing `Updated` property in the `Ebook` class to `MetadataModified`. I chose the name based on how the existing property is [set](https://github.com/standardebooks/web/blob/ff8cc6554d04c0eed172329d9c9f5f33aefe8f02/lib/Ebook.php#L228):

```php
		$modifiedDate = $xml->xpath('/package/metadata/meta[@property="dcterms:modified"]') ?: [];
		if($modifiedDate !== false && sizeof($modifiedDate) > 0){
			$this->Updated = new DateTime((string)$modifiedDate[0]);
		}
```

The full set of name changes are:

* Rename `Created` to `Released`
* Rename `Updated` to `MetadataModified`

in order to add new `Created` and `Updated` properties in the `Ebook` class that map to the columns in the `Ebooks` table.

Reply:
OK I see. Let's tweak this in the following way: `Created` and `Updated` will be the timestamps for the DB row, so it's consistent with the rest of the DB. Then let's rename these two to `EbookCreated` and `EbookUpdated`, so it's clear they're related and that it's a property linked to the ebook itself and not the DB row, but is also consistent with how we name the columns otherwise.

Reply:
Works for me. Done.